File: WebService/src/MathClient.py
import suds # pip install suds

class MathClient:
    def __init__(self):
       self.client = suds.client.Client("http://localhost:9999/ws/marita?wsdl")
    
    def sum(self, a, b):
       return self.client.service.sum(a,b);

    def sub(self, a, b):
       return self.client.service.sub(a,b);
       
    def mult(self, a, b):
       return self.client.service.mult(a,b);
       
    def div(self, a, b):
       return self.client.service.div(a,b);              

    def fak(self, a):
       return self.client.service.fak(a);

    def fib(self, a):
       return self.client.service.fib(a);

    def ack(self, a, b):
       return self.client.service.ack(a,b);

if(__name__ == "__main__"):
    client = MathClient()
    print client.sum(6,3)
    print client.sub(19,2)
    print client.mult(6,3)
    print client.div(19,4)
    print client.fak(5)           
    print client.fib(8)
    print client.ack(3,2)    

File: WebService/src/client/MathClient.java
package client;

import java.net.MalformedURLException;
import java.net.URL;

import javax.xml.namespace.QName;
import javax.xml.ws.Service;

import ws.MyMath;

// client for testing. we import MyMath directly without generation service class using wsimport

public class MathClient {
	public static void main(String[] args) throws MalformedURLException {
		URL url = new URL("http://localhost:9999/ws/marita?wsdl");
		
	    //1st argument service URI, refer to wsdl document above
		//2nd argument is service name, refer to wsdl document above
		QName qname = new QName("http://ws/", "MathImplService");
	
        Service service = Service.create(url, qname);

        MyMath math = service.getPort(MyMath.class);

        System.out.println(math.sum(3, 6));
        System.out.println(math.sub(3, 6));
        System.out.println(math.mult(3, 6));
        System.out.println(math.div(13, 6));
        System.out.println(math.fak(6));
        System.out.println(math.fib(6));        
        System.out.println(math.ack(3,2));
	}
}
